python/gsw/endpoint_configs/telemetry_config.yml

```diff
@@ -0,0 +1,28 @@
+summary: "Send telemetry data to the ElasticSearch server in field-value-time tuples"
+tags: 
+  - "Post Telemetry Data to ElasticSearch"
+description: "This is used for sending and indexing telemetry data to the ElasticSearch server. Note that this will not generate an iridium report"
+consumes:
+  - "application/json"
+produces:
+  - "application/json"
+parameters:
+  - in: "body"
+    name: "body"
+    description: "Accepts a Field-Value-Time Tuple"
+    required: true
+    schema:
+      type: "object"
+      properties:
+        time:
+          type: "string"
+          example: "11/12/13"
+        field:
+          type: "string"
+          example: "control cycle count"
+        value:
+          type: "integer"
+          example: 6
+responses: 
+  405:
+    description: "Invalid input"
```

Comment:
When do you return 405 if the input is invalid? I couldn't find it in the code, I probably just missed it though

Reply:
The endpoint already returns a different error message when an index or field can't be found, so yeah this doesn't really do anything.